core/src/main/java/io/aiven/klaw/service/ClusterApiService.java

```diff
@@ -447,6 +432,10 @@ public String approveConnectorRequests(
 
     } catch (Exception e) {
       log.error("approveConnectorRequests {} ", connectorName, e);
+      if (e.getMessage().contains("ClientHttpRequestFactory must not be null")
+          || e.getMessage().contains("Connection refused")) {
```

Comment:
"Connection refused" && "ClientHttpRequestFactory must not be null" are re-used here so you could make final statics in this class.